types/node/index.d.ts

```diff
@@ -797,82 +809,93 @@ declare module "http" {
     /**
      * @deprecated Use IncomingMessage
      */
-    export interface ServerRequest extends IncomingMessage {
+    export class ServerRequest extends IncomingMessage {
         connection: net.Socket;
     }
 
-    export interface ServerResponseHeaders {
-      [key: string]: number | string | string[];
-    }
+    // https://github.com/nodejs/node/blob/master/lib/_http_outgoing.js
+    export class OutgoingMessage extends stream.Writable {
+        output: any[];
```

Comment:
Not sure if it is a good idea to add non documented properties here which were not yet here before as they may change even on patch version of NodeJs.